### PR TITLE
Ml4 patch 5

### PR DESCRIPTION
**Pull request by ml4**
I appreciate this may not fully fix the link issue discussed, but I'm a rebase amateur, and it's taken me about 90 mins just to get the branch from my fork rebased and the PR raised.  I need to read the git manual more thoroughly.

### CODE REVIEW

**Comment by ml4** [comment]
...and still a merge conflict resulted, even though I'd done this prior to rebasing and pushing to origin master.  I had to use GH UI to create the PR (which is where I smelt a rat) as one wasn't offered when I pushed to my fork's master.

File: website/pages/docs/job-specification/service.mdx [hunk 1]
```diff
@@ -705,7 +705,7 @@ advertise and check directly since Nomad isn't managing any port assignments.
 </sup>
 <small>
   {' '}
-  Script checks are not supported for the [QEMU driver][qemu] since the Nomad
+  Script checks are not supported for the [qemu] driver since the Nomad
```
Comment (tgross) [inline website/pages/docs/job-specification/service.mdx:708]:
```suggestion
  Script checks are not supported for the QEMU driver since the Nomad
```

**Review by tgross — CHANGES_REQUESTED** [review]
Hi @ml4! One last item and then if you accept that suggestion this will LGTM and we can merge it.

Next time let me know if you're running into issues and I can help walk you thru!

**Comment by ml4** [comment]
Thanks Tim - I was using it as an excuse to (a) force myself to try myself to learn and (b) not ask you for help given I havent read enough of the git documentation.  I get rebasing now though 👍🏼

**Review by tgross — APPROVED** [review]

**Pull request merged**